sfcasts/crud-controller.md

```diff
@@ -1,171 +1,90 @@
-to our other sections, it's still there.
-
-So the big takeaway here is that everything is going through our dashboard
-controller, which means that we can configure things on a dashboard level, which will
-apply to all of our crus, or we can configure things for one specific crud. Now, the
-fact that all of the crud controllers go through this /avenue, where all has one
-other effect related to security, it means that all of our controllers are already
-secure that's thanks to our access control. So remember back in config packages
-secure to that YAML, we had a little access control here that said if the URL starts
-to /admin require roll_admin. So all of these are automatically now without doing
-anything else, requir role admin, we'll talk more later about how to secure different
-admin controllers with different roles, but at the very least, you have to have role
-admin to get anywhere, which is awesome. But one important point, adding this access
-control was necessary. Here's a, even though the URL is /admin, when you go to a crud
-control like this, it does not exit cute our dashboard controller index action. So
-this is a little bit confusing.
-
-This action here is what has our route.
-
-When we go to a crowd controller like this, it does match this route, but easy admin
-does something crazy. Instead of allowing Symfony to call this controller. It sees
-these crud, this crud controller query parameter here and magically switch is the
-controller to be the real controller. In this case, to be question crowd controller
-index, you can actually see this down here on the web Depot toolbar. If you hover
-over these a admin at admin, tell this tells you that the route name is admin. So the
-route name is actually this route right here, but the controller is question crud
-controller, call on call an index. So there's some route magic happening, but
-ultimately the methods in your crud controller are the ones that are called. So it's
-the index method in this abstract crud controller down here. This is actually the
-real roller for the page. This is important for security, just because if we had only
-put the route and the route attribute above index and not added and not added the
-access control, that wouldn't have been enough
+# CRUD Controller
+
+The true reason you use easy admin is for its CRUD controllers. Each CRUD controller will give us a rich set of pages to create, read, update, and delete a single entity. This is where EasyAdmin shines, and the next few minutes are going to be critically important to understanding how EasyAdmin works.
+
+We have four entities, so let's generate a CRUD controller. We'll do that for `Question` first. Find a terminal and run:
+
+```
+symfony console make:admin:crud
+```
+
+As you can see, it recognizes our four entities. I'll hit 1 for `App/Entity/Question`. Let this generate into the default directory and with default namespace.
```

Comment:
Should this be "generate into" or just "generate"? This is a tiny technical detail I wasn't really sure about.